extras/module-node/Makefile

```diff
@@ -14,3 +14,4 @@ test:
 	./test 'var ape = require("ape"); ape.module.filename === "ape.js" ? "module.filename OK" : "module.filename FAILED";'
 	./test 'var ape = require("ape"); ape.module.loaded === true && ape.wasLoaded === false ? "module.loaded OK" : "module.loaded FAILED";'
 	./test 'var ape = require("ape"); ape.__filename === "ape.js" ? "__filename OK" : "__filename FAILED";'
+	./test 'var badger = require("badger"); badger.foo === 123 && badger.bar === 234;'
```

Comment:
Not sure what the purpose of this test is?  It will throw since everything other the pig/cow/ape exports undefined.